Add KOTS security context (#2492)
replicatedhq/replicated-docs: docs/reference/kots-cli-admin-console-generate-manifests.mdx

import KotsadmRegistry from "../partials/kots-cli/_kotsadm-registry.mdx"
import RegistryPassword from "../partials/kots-cli/_registry-password.mdx"
import RegistryUsername from "../partials/kots-cli/_registry-username.mdx"
import StrictSecContext from "../partials/kots-cli/_strict-sec-context-yaml.mdx"

# admin-console generate-manifests

Running this command will create a directory on the workstation containing the Replicated Admin Console manifests. These assets can be used to deploy KOTS to a cluster through other workflows, such as kubectl, to provide additional customization of the Admin Console before deploying.

### Limitations

* `generate-manifests` does not support generating manifests for Red Hat OpenShift clusters or GKE Autopilot clusters if executed without a Kubernetes cluster context.

* To upgrade a KOTS instance that has ever been on version 1.72.0 or earlier, you must run `generate-manifests` with a Kubernetes cluster context.

> * The `admin-console generate-manifests` command does not accept the [`--strict-security-context`](/reference/kots-cli-install#usage) flag, which deploys KOTS Pods and containers with a strict security context. To generate Admin Console manifests with a strict security context, add the following to the kotsadm Deployment spec:

Author: >add the following to the kotsadm Deployment spec:

This was my guess as to where to add it...If this is accurate, would it be in the `containers` field of the deployment spec? Or maybe they would just be adding it to their own cluster Pod specs?

Reviewer: You have to add this to kotsadm, kotsadm-minio, and kotsadm-rqlite. But if you're on an older version or are excluding MinIO, it might be different. So I don't know whether we should list these as examples, but I think a vendor knowledgeable about this can figure it out.
```suggestion
* The `admin-console generate-manifests` command does not accept the [`--strict-security-context`](/reference/kots-cli-install#usage) flag, which deploys KOTS Pods with a security context. To generate Admin Console manifests with a security context, add the following to the Pod templates for Deployments and StatefulSets deployed by KOTS:
```

Author: Gotcha, thanks
